plugins/modules/ovirt_vm.py

```diff
@@ -914,6 +914,14 @@
             - "If `true`, each virtual interface will get the optimal number of queues, depending on the available virtual Cpus."
         type: bool
         version_added: 1.7.0
+    virtio_scsi_multi_queues:
+        description:
+            - "Number of queues for a Virtio-SCSI contoller, possible values:
+              -1 - Indicates that the queues will be automatically set, see virtio_scsi_enabled
```

Comment:
"see virtio_scsi_enabled" should be dropped

Reply:
Fixed